src/Core/TestBootstrapper.php

```diff
@@ -378,6 +378,10 @@ private function installPlugins(): void
                 'plugins' => [$activePlugin],
             ];
 
+            if (array_key_exists('command', $installCommand->getDefinition()->getArguments())) {
+                $args['command'] = 'plugin:install';
```

Comment:
What happens if you add this always?

Reply:
and this sounds really like a bug in Symfony wtf

Reply:
Its a quite weird behaviour. In the first run the Definition arguments look like this:

![2022-10-11 at 09 47](https://user-images.githubusercontent.com/10775702/195028996-25832d97-d80a-4fba-83c8-1455b990df78.png)

For the second plugin / loop element:

![2022-10-11 at 09 48](https://user-images.githubusercontent.com/10775702/195029401-25ab6e10-9cb4-4c48-a23c-5aa7b686b80f.png)


So when I add it in the first run it throws:

```
Error in bootstrap script: Symfony\Component\Console\Exception\InvalidArgumentException:
The "command" argument does not exist.
#0 /Users/florianliebig/code/shopware-platform/vendor/symfony/console/Input/ArrayInput.php(141): Symfony\Component\Console\Input\ArrayInput->addArgument('command', 'plugin:install')

```

I just figured out that it works to introduce a new variable for the definition:

![2022-10-11 at 09 51](https://user-images.githubusercontent.com/10775702/195030642-57a642c8-90c6-408a-ba59-1d0ed8a69418.png)

This seems like a cleaner solution to me

Reply:
Ahh okay. Maybe worth creating a issue at symfony repository too

Reply:
https://github.com/symfony/console/blob/6.1/Command/Command.php#L377

It seems to get overwritten by the fullDefinition (which has two arguments) here

Reply:
https://github.com/symfony/console/blob/6.1/Command/Command.php#L299

Nevermind it's that line